Week1/homework/todo sql.sql

```diff
@@ -0,0 +1,42 @@
+CREATE DATABASE IF NOT EXISTS  base_todoinfo;
+
+USE base_todoinfo;
+
+CREATE TABLE IF NOT EXISTS todos(
+	id_todo INT NOT NULL,
+    todo_description VARCHAR(255),
+    dead_line DATE,
+    done BOOLEAN,
+    PRIMARY KEY(id_todo)
+);
+CREATE TABLE IF NOT EXISTS users(
+	id_user INT NOT NULL,
+    name VARCHAR(25),
+    PRIMARY KEY(id_user),
+    todos_id_todo INT NOT NULL,
+    CONSTRAINT fkuser_todos
```

Comment:
this constraint here implies the relation between users and todos is one user has one todo and one todo has many users 🤔  If you did this one from a sql generatior make sure the `many` sides of the relationship chicken wings is pointing to the correct table

Reply:
ohh yes sorry, the idea from the start was to make one user to have many todos, u put the fk in the wrong place :)